app/javascript/app/components/ghg-emissions/ghg-emissions.js

```diff
@@ -183,7 +183,7 @@ class GhgEmissionsContainer extends PureComponent {
     const { source } = this.props.sourceSelected;
     if (source) {
       this.props.setModalMetadata({
-        slug: source,
+        slugs: [source],
```

Comment:
I'd rather prefer to check the method if is a string or an array than set an array as mandatory param.